aws/README.md

```diff
@@ -100,3 +101,31 @@ Assumes the following are installed through choco:
 ```
 tag_instance.ps1
 ```
+
+## update_other_tag.py
+
+Update the value of CostCenterOther for a specified EC2 instance, including related resources in the same cloudformation stack and/or service catalog product.
```

Comment:
Can you please provide info on why this was created and what use case it solves? maybe a reference to a jira issue might help?

Also it would be good to know whether this script is doing a replace or additive operation.